.gitignore

```diff
@@ -100,3 +100,4 @@ lib/**/*
 
 # Ignore output files
 dist
+example/action-dist
```

Comment:
Nope! I don't expect others to actually use this action. It's useful for dog-fooding within the repo itself, both as a check that nothing breaks and a demonstration of full use. We run this action on every push ([source](https://github.com/github/dependency-submission-toolkit/actions/workflows/dependency-submission-example.yml)). Runs are here: https://github.com/github/dependency-submission-toolkit/actions/workflows/dependency-submission-example.yml

Reply:
You can see that action successfully completing on this PR here: https://github.com/github/dependency-submission-toolkit/actions/runs/2557521579 . We're just building it before running to avoid checking in dist, that's happening [here](https://github.com/github/dependency-submission-toolkit/pull/15/files#diff-36390fd1fff6fea96e8c3c3999c3d6a15f077fde9190628e5811505df2217b6bR14)

Reply:
OK, so just clarifying you want to leave the action-dist in source control because a real life implementer of an action would need to do the same thing? I can back out that change if that's the case